### PR TITLE
Ducati rental V1.0

### PR DESCRIPTION
**Pull request by jevazquezb**
## Ducati Rental V1.0

This application allows the user to reserve one of our motorcycle models once they have logged in. Also, the user is able to add another model to our database and reserve that specific model to get a test ride. Finally, the user can delete one of the motorcycle models if it is not available anymore.

To built this app we followed the following requirements:

**Basics**
- Followed the layout provided by Murat Korkmaz on [behance](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign).

**Core Features**

- Added a login/signup page for the user to register.
- Implemented a navigation menu where the user can access the following sections:
  - Models.
  - Reserve.
  - My reservations.
  - Add motorcycle.
  - Delete motorcycle.
  - Log out.
- Added a list of the motorcycle models on the main page.
- Added a "details" page to see the specifications of each motorcycle.
- Integrate a motorcycle form to add a new model to the database.
- Added a "delete" page where the user can delete a motorcycle model.
- Implemented a "reservations" page where the user can rent a bike model.
- Added a "my-reservations" page where the user can see its rental list.
- Built the mobile and desktop versions of the app.
- The App is connected to an [API](https://bikes-db.onrender.com/) built with Ruby on Rails & PostgreSQL: [Back-end repository](https://github.com/AbrahaKahsay/final-capstone-backend).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+### Install
+
+    In the terminal, navigate to where you wish the repo to be located and run these commands.
+
+```
+$ git clone https://github.com/AbrahaKahsay/final-capstone-frontend.git
+```
+```
+$ cd final-capstone-backend
```
Comment (Meltrust) [inline README.md]: - [x] This dir should be final-capstone-frontend if we are trying to setup the frontend to run.  Kindly correct it.

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { Routes, Route } from 'react-router-dom';
+import Main from './pages/main';
+import Authentification from './pages/authentification';
+
+function App() {
```
Comment (Meltrust) [inline src/App.js]: - [x] Please, kindly refactor this functional component  to ES6 arrow function, so your code complies with the latest standards.

The function keyword has been soft deprecated all over the software industry.  This means, almost nobody uses it anymore, so much that it's considered non best practice.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry.

image

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @jevazquezb,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation, but you are almost there!






### To highlight:


-  App is is working well✔️ 
-  Good readme ✔️ 
-  Beautiful frontend ✔️ 



#### You are really close to finishing the Microverse program!! Keep it up! 👍👍👍

You can do it

### After implementing the requested changes, please submit another review request. ♻️
_Check the comments under the review._

_I strongly recommend you take them into account as they can improve your final evaluation._





Cheers and Happy coding!👏👏👏

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the previous reviews unless it is requested otherwise._**

------

File: README.md (hunk 1 above)
Comment (lucas-crodrigues) [reply README.md]: Will do thanks

File: src/App.js (hunk 2 above)
Comment (lucas-crodrigues) [reply src/App.js]: You're right, thank you

**Comment by Meltrust** [comment]
#### Hi @jevazquezb,


# **Wow, you did it 🎉**
Brilliant
## Thank you for the changes implemented 💪 🥇 ㊗️ 
#### _Unless you want to add more features, go ahead to your final presentation ⏩ ⏩ ⏩_ 
## You are about to finish the Microverse program.  You have come a long way!!!




### Good luck in the software industry!!  I'll see you there. ✨
### **Congratulations!!!!!!**  🎉
applause

### To highlight

- Good job!!! ✔️ 




Cheers and Happy coding!👏👏👏


------

**Pull request merged**